backend/src/main/java/com/bakdata/conquery/sql/conversion/model/filter/BigMultiSelectFilterConverter.java

```diff
@@ -0,0 +1,12 @@
+package com.bakdata.conquery.sql.conversion.model.filter;
+
+import com.bakdata.conquery.models.datasets.concepts.filters.specific.BigMultiSelectFilter;
+import com.bakdata.conquery.sql.conversion.cqelement.concept.FilterContext;
+
+public class BigMultiSelectFilterConverter extends AbstractSelectFilterConverter<BigMultiSelectFilter, String[]> {
+
+	@Override
+	protected String[] getValues(FilterContext<String[]> filterContext) {
```

Comment:
die abstraktion musst du gar nicht machen, Big/-Multi sind fürs FE relevant und ein bisschen, wie wir sie aufbereiten dafür im Backend. Logik ist die gleiche

Reply:
Was meinst du mit "muss ich nicht machen?" Heißt das im Backend kommt immer nur ein SelectFilter und für Big/Multi brauch ich keinen Converter?

Reply:
sry habe das nicht in meinen PRs gesehen. du kannst für beide den gleichen converter verwenden.

Kurze warunung, ich habe vor die FilterValue auf Set<String> zu migrieren. Das nicht zu machen hat bei uns zu performance problemen geführt.